rotkehlchen/db/dbhandler.py

```diff
@@ -130,6 +131,27 @@ def db_tuple_to_str(
     raise AssertionError('db_tuple_to_str() called with invalid tuple_type {tuple_type}')
 
 
+class DBSettings(namedtuple):
+    version: int
+    last_write_ts: int
```

Comment:
This should be of type `Timestamp`

Reply:
Are you sure about this?
```
elif q[0] == 'last_write_ts':
    settings['last_write_ts'] = int(q[1])
```
This was the code before I changed it and here we have an int

Reply:
Yeah the code was wrong before but good point. I fixed it some subsequent commits.